modules/serverpod_auth2/serverpod_auth_user/serverpod_auth_user_client/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Files and directories created by pub
+    .dart_tool/
```

Comment:
This extra padding seems to have snuck into the template. I can fix it there and in all affected packages in a follow-up :)

Reply:
Done in https://github.com/serverpod/serverpod/pull/3461